doc/interfaces/supercomputer_data/drafts/20231009_region_hierarchy/root_regions.json

```diff
@@ -0,0 +1,37 @@
+{
+    "26000": {
+        "name": "Western United States",
+        "code": "USwest",
+        "subdivisionType": "adm0group",
+        "EPSG": 3857,
+        "subdivisionListAndMetadataRelativePath": "subdivisionlist/subdivisionlist_26000.json"
+    },
+    "26002": {
+        "name": "ASHimalayas",
```

Comment:
What does "AS" stand for here? For the `"name"` field that we're going to show to users, let's spell it out!

Reply:
AS is for Asia, AM for America. I propose to let Karl be in charge of deciding the final names, to avoid spending time on it (even if very small time)